package.json

```diff
@@ -5,22 +5,24 @@
   "author": "Andrew Harmel-Law",
   "license": "CC-BY-NC-SA-4.0",
   "dependencies": {
-    "skill-scm-distributed-git-apprentice": "github:andrewharmellaw/skill-scm-distributed-git-apprentice",
-    "skill-networking-apprentice-dev": "github:andrewharmellaw/skill-networking-apprentice-dev",
-    "skill-unix-apprentice-user": "github:andrewharmellaw/skill-unix-apprentice-user",
-    "skill-ooad-apprentice-dev": "github:andrewharmellaw/skill-ooad-apprentice-dev",
+    "doc-publisher": "^0.1.15",
```

Comment:
Yep makes sense - think that was a missed step on my behalf. Runs fine when moving the 2 dependencies into devDependencies, so I can remove the skill filtering and just grab the dependencies as is (as they'll just be skills) and publish that way.

Not sure how to remove the publish-to-local-docs entirely and call from the travis build though? Would it in some sort of script?